onnxruntime/hosting/http/predict_request_handler.cc

```diff
@@ -1,35 +1,64 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+#include <google/protobuf/stubs/status.h>
+
 #include "environment.h"
 #include "http_server.h"
 #include "json_handling.h"
+#include "executor.h"
+#include "util.h"
 
 namespace onnxruntime {
 namespace hosting {
 
+namespace protobufutil = google::protobuf::util;
+
+#define GenerateErrorResponse(logger, error_code, status, context)                   \
+  {                                                                                  \
+    auto http_error_code = (error_code);                                             \
+    auto error_message = CreateJsonError(http_error_code, (status).error_message()); \
+    LOGS((*logger), VERBOSE) << error_message;                                       \
+    (context).response.result(http_error_code);                                      \
+    (context).response.body() = error_message;                                       \
+    (context).response.set(http::field::content_type, "application/json");           \
+  }
+
 // TODO: decide whether this should be a class
 void Predict(const std::string& name,
              const std::string& version,
              const std::string& action,
-             HttpContext& context,
+             /* in, out */ HttpContext& context,
```

Comment:
I don't think this needs to be stated

Reply:
Just try to be clear to the code reader.